### ISSUE TITLE
Accessing Tizen API

### ISSUE THREAD

**Issue by KuluGary** [pavjacko/renative#339]
Hello. I have been working in a cross platform that includes, aside from others, the Tizen TV platform. I am interested in being able to download some information into the device in order to cache content.

I have been trying to use the different Tizen APIs such as [Filesystem API](https://www.tizen.org/ko/tv/web_device_api/filesystem?langredirect=1) and [SystemInfo API](https://www.tizen.org/ko/tv/web_device_api/systeminfo?langredirect=1). However, whenever I try to use something like `console.log(tizen.systeminfo)` it doesn't allow me to, even though I found this in this github [https://github.com/pavjacko/renative/issues/225](#255).

The example I am trying to use and can't right now because it throws 'detailReferenceError: tizen is not defined'.
```
import Configuration from '../../Helpers/Configuration';
import { Api } from 'renative';

export default class PersistentStorage {
    constructor() {
        this.filesystem = tizen.filesystem;
    }

    initialize() {
        return new Promise((resolve, reject) => {
            this.filesystem.createDirectory('documents/cache', resolve(true), reject(false));
            this.internalPath = 'documents/cache';
        })
    }
}
```

How would one go about accessing these platform specific APIs?

**Comment by KuluGary** [comment]
Hello! I've  since continued working on the project mentioned in my original post, but this bug has kept me blocked in the development of Tizen related components.

Is there any way I can help in terms of fixing this bug, be it assisting with information on its nature or coding? If not, do we have any estimated time in which we could have some news about the issue?

Thanks, I'll be waiting for any kind of response!

**Comment by mihaiblaga89** [comment]
We're still working on this, I don't have a timeline for it yet. I hope to get it fixed by the end of the week

**Comment by KuluGary** [comment]
Hello! I've still been struggling with this issue. I'm sorry to be insistent on this, but it has stopped my progress developing the app. Is there any new information regarding this bug that the team can share? Any way I can help offering support to solve the problem?

Thank you again.

**Comment by mihaiblaga89** [comment]
@KuluGary as a status update and to unblock your project, you can access tizen's object by running the app with `"bundleAssets": true` in your scheme. The problem with the object is only present when running in dev mode with webpack

**Comment by mihaiblaga89** [comment]
@KuluGary sorry for the delay of the reply, this was fixed in #351
